libraries/c_sdk/aws/defender/test/system/aws_iot_tests_defender_system.c

```diff
@@ -76,6 +79,23 @@
 /* Use a big number to represent no event happened in defender. */
 #define NO_EVENT                             10000
 
+/**
+ * @cond DOXYGEN_IGNORE
+ * Doxygen should ignore this section.
+ *
+ * Provide default values of test configuration constants.
+ */
+#ifndef IOT_TEST_MQTT_CONNECT_INITIAL_RETRY_DELAY
+    #define IOT_TEST_MQTT_CONNECT_INITIAL_RETRY_DELAY    ( 1100 )
+#endif
+#ifndef IOT_TEST_MQTT_CONNECT_RETRY_COUNT
+    #define IOT_TEST_MQTT_CONNECT_RETRY_COUNT            ( 1 )
```

Comment:
I'd suggest a higher count here, for example 6 ([the value used in TCP tests.](https://github.com/aws/amazon-freertos/blob/master/tests/include/aws_test_tcp.h#L58)).